examples/Attributes/publications.md

```diff
@@ -0,0 +1,113 @@
+the `biolink:publications` property as the key attribute_type_id, as defined below:
+
+```yaml
+publications:
+  aliases: ["supporting publications", "supporting documents"]
+  is_a: association slot
+  description: >-
+    One or more publications that report the statement expressed in an Association, 
+    or provide information used as evidence supporting this statement. 
+    The notion of a ‘Publication’ is considered broadly to include any document made   
+    available for public consumption. It covers scientific journal issues, individual articles, and
+    books - as well as things like pre-prints, white papers, patents, drug
+    labels, web pages, protocol documents, and even a part of a publication if
+    of significant knowledge scope (e.g. a figure, figure legend, or section
+    highlighted by NLP).
+  range: publication
+```
+
+Note, the range for `biolink:publications` is `biolink:publication` which is actually a [Biolink Model class](https://biolink.github.io/biolink-model/docs/Publication.html#class-publication).  However, using a collection of CURIEs as shorthand syntax for representing a full list of `biolink:Publication` objects 
+is expected in a TRAPI message (see below).
+
+
+### Implementation Guidance
+
+1. The `biolink:publications` edge property MUST be used as the `attribute_type_id` in Attributes reporting publications that support an Edge.
+
+```json
+"attribute_type_id": "biolink:publications"
+```
+
+2. Where multiple distinct publications support a single Edge, these MAY be reported together in a single Attribute object 
+as a list in the `value` field, as below:
+
+```json
+"value": ["PMID:31737390", "PMID:6815562","http://info.gov.hk/gia/general/201011/02/P201011020204.htm"] 
+```
+
+3. Supporting publications are most typically referenced using a CURIE or URI/URL (as above), but may also be captured as free-text strings that describe the publication.  
+
+4. If a source provides an Edge where some publications are referenced as CURIE/URIs, and other reported as free-text descriptions for others - the publications referenced by CURIE or URI MUST be captured in a separate Attribute object from those referenced as free-text.  
+     a. The `value_type_id` in the Attribute holding CURIEs and/or URIs MUST be `biolink:uriorcurie`  
+     b. The `value_type_id` in the Attribute holding free text descriptions MUST be **[T.B.D.]**   
+
+```json
+{
+  "edges": [
+    {
+      "id": "Association001",
+      "subject": "CHEBI:3215",
+      "predicate": "biolink:interacts_with",
+      "object": "NCBIGene:51176",
+      "attributes": [
+        {
+          "attribute_type_id": "biolink:publications",
+          "value": [
+            "PMID:31737390",
+            "PMID:6815562",
+            "http://info.gov.hk/gia/general/201011/02/P201011020204.htm"
+          ],
+          "value_type_id": "biolink:Uriorcurie",
```

Comment:
@mbrush @sierra-moxon In this guide you used `"value_type_id": "biolink:Uriorcurie"` , however according to Biolink it should be `"value_type_id": "metatype:Uriorcurie"` (https://biolink.github.io/biolink-model/docs/types/Uriorcurie.html). I prefer `biolink:Uriorcurie` though.